Simple-PHP-IPAM/lib.php

```
@@ -2042,39 +2314,27 @@ function ipam_config_sync(string $configPath, array $loaded): array
-    }
-
-    $auditRetention = to_int($config['audit_log_retention_days']);
-    if ($auditRetention < 0) {
-        $warnings[] = "audit_log_retention_days is {$auditRetention}; must be 0 or greater.";
+    $checks = [
+        ['security.session_idle_seconds',  60, '>=', 'session_idle_seconds'],
+        ['security.login_max_attempts',     1, '>=', 'login_max_attempts'],
+        ['security.login_lockout_seconds',  1, '>=', 'login_lockout_seconds'],
+        ['housekeeping.audit_log_retention_days', 0, '>=', 'audit_log_retention_days'],
+    ];
+    foreach ($checks as [$key, $min, $op, $label]) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$op'. (undefined)

(UnusedLocalVariable)